src/components/Beneficiaries/index.tsx

```diff
@@ -123,6 +128,10 @@ class BeneficiariesComponent extends React.Component<Props, State> {
         if (nextProps.beneficiariesActivateSuccess && !beneficiariesActivateSuccess) {
             this.handleToggleConfirmationModal();
         }
+
+        if (nextProps.beneficiariesDeleteSuccess && !beneficiariesDeleteSuccess) {
+            this.props.beneficiariesFetch();
+        }
```

Comment:
currently WalletsScreen includes handler for BeneficiaryDelete ( see image)
![image](https://user-images.githubusercontent.com/39953699/105679508-3be92400-5ef7-11eb-96d0-35a23289299f.png)

Reply:
Besides, beneficiary flow has been updated. Now we fetch beneficiary only for selected wallet

Reply:
Fixed, remove extra logic in Beneficiaries component. thanks.